pkg/config/dynamicloader_test.go

```diff
@@ -0,0 +1,759 @@
+package config //nolint:golint
```

Comment:
Going to look through this today and see what I can move into functions to deduplicate some code.